src/tmdb.py

```diff
@@ -339,7 +339,6 @@ async def tmdb_other_meta(
     tmdb_type = ""
     mal_id = 0
     demographic = ""
-    logo_path = None
```

Comment:
It needs to be set empty here also, else if a user does not have the config option set, it will not iterate through that code path and error.

Reply:
it is set a few lines above this one

Reply:
ah, so it is.